publish-release.md

```diff
@@ -0,0 +1,10 @@
+# Publish Release
+
+## Goals
+- Require Git branch is main
+- Require Git workspace is clean
+- Update version field in all `package.json` files
+- Create release commit (commit can be empty with `--allow-empty`)
+- Git tag release commit with semantic version
+- npm release tag with semantic version.
+- Create GitHub Release. Set name of the release to the value of semantic version and publish.
```

Comment:
maybe then smoke test the packages (e.g. in the client, using the `test-exports`), or is this next-version feature? :) Of course that smoke-test could and maybe should happen already during the CI test

Reply:
CI does it already.